### DISCUSSION TITLE
Can't connect to Spotify there is no sp_dc cookie

### DISCUSSION THREAD

**Discussion by controlunit** [KRTirtho/spotube#1472] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

sp_dc cookie not present

### Expected Behavior

presence

### Steps to reproduce

Step 3 in connecting to Spotify

![sp_dc_not_there](https://github.com/KRTirtho/spotube/assets/63740471/ce1510b0-03a4-4d85-8f68-31ac4d01ee29)

### Operating System

Yandex browser

### Spotube version

Kingkor Roy Tirtho    com.github.KRTirtho.Spotube   3.5.0 

### Installation source

Flathub (Flatpak)

### Self grab

- [ ] I'm ready to work on this issue!

**Comment by Obambulate** [comment]
could you tell me your process in finding the page you are in to find these cookie listings?

(If you press ctrl+shift+i a window pops up then go to the application tab and find where is says Cookies, on the left, open that and the cookie should be in there)

**Comment by KRTirtho** [comment]
You've to be logged in with your Spotify Account first in the browser. `sp_dc` becomes available only when user is logged in
